src/Contract/Time.purs

```diff
@@ -97,11 +102,75 @@ import Ctl.Internal.Types.Interval
   ) as Interval
 import Ctl.Internal.Types.SystemStart (SystemStart)
 import Ctl.Internal.Types.SystemStart (SystemStart(SystemStart)) as ExportSystemStart
+import Data.Array as Array
 import Data.BigInt as BigInt
+import Data.Foldable (find)
+import Data.Maybe (Maybe(Just, Nothing))
+import Data.Newtype (unwrap)
 import Data.UInt as UInt
+import Effect.Aff (delay)
 import Effect.Aff.Class (liftAff)
 import Effect.Exception (error)
 
+-- | Get a summary of the current era.
+getCurrentEra :: Contract ExportEraSummaries.EraSummary
```

Comment:
Import it twice. Please use `import ... as X` for imports that are meant to be re-exports. See https://github.com/Plutonomicon/cardano-transaction-lib/issues/56 (please change other re-exports as well - and change if there are duplicates with other `Contract.*` modules)